source/docs/software/wpilib-overview/3rd-party-libraries.rst

```diff
@@ -83,3 +83,10 @@ Removing a Library Dependency
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 To remove a library dependency from a project, select **Manage Current Libraries** from the **Manage Vendor Libraries** menu, check the box for any libraries to uninstall and click OK. These libraries will be removed as dependencies from the project.
+
+Command-Line Interface (Online)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Adding a vendor library dependency from the vendor URL can also be done through the command-line via a gradle task. Open a command-line instance at the project root, and enter `gradlew vendordep --url=<url>` where `<url>` is the vendor JSON URL. This will add the vendor library dependency JSON file to the **vendordeps** folder of the project. Vendor libraries can be updated the same way.
+
+.. note:: Installing vendor libraries via the command-line will only add the library to the current project, and will not add the library to the general `wpilib` folder. A `build` operation while online will add the library to the general `wpilib` folder, regardless of the library installation method.
```

Comment:
This sentence is confusing and I have very little idea what this means. "Adding it to the general 'wpilib' folder"? I assume you mean it downloads the dependency for usage, however the text makes it assume that the vendordep will be available to all new projects

Reply:
I meant download the dependency to the local frcmaven repo for usage, I just misphrased the sentence.

Reply:
Now that I think about it, is the note needed? It is no different than the VSC online method, and the note simply reiterates a point that was covered already.

Reply:
I don't think the note is necessary. Agreed.